libs/mgjson/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mgjson
+PKG_RELEASE:=1
+
+PKG-BRANCH=main
+PKG_SOURCE_URL=https://github.com/TheRootED24/mgjson.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2024-08-17
+
+PKG_SOURCE_VERSION:=9793e7ba97f1197909dda8fa3002673e4dd7266b
+PKG_MIRROR_HASH:=85c5b3c3704e0b834f4d315b345baadd79a5e972bf688f46269d2887d4ae6acd
+
+PKG_MAINTAINER:=Scott Mercer <TheRootED24@gmail.com>
+
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/mgjson
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=A Stripped down OpenWrt version of Mongoose C Networking. mgjson provides only the mg_json, mg_str, and mg_iobuf, functionilty of the Mongoose Library.
```

Comment:
How did you tested it? Have you checked ``make menuconfig``?

Reply:
yes, all build and basic functionality have been tested thoroughly. 

![Screenshot from 2024-08-21 13-05-00](https://github.com/user-attachments/assets/d902d418-1dc7-4e13-9b06-efb28d5401e6)
![Screenshot from 2024-08-21 13-03-35](https://github.com/user-attachments/assets/e1349eca-4a8c-4017-ba96-6fc39f95386e)

I never included a description for the mgjson library, this should probably be included ?

Reply:
You shared two screenshots. The first one even shows that something is wrong. Title is not shown as it is too long.

Reply:
yes, I mentioned that in the comment attached to the photos. This was an oversight on my part, I forgot to add a description field. This library, as you mentioned, is only 3 weeks from conception. I probably should've have started with the more mature library first. In any case. i will add a the description field.

Reply:
If the title is to long, I am fine trimming it to the period ie.
```
A Stripped down OpenWrt version of Mongoose C Networking.
```